packages/core/src/fetchers/fetchAllEntities.spec.ts

```diff
@@ -33,17 +43,25 @@ describe('fetchAllEntries', () => {
       limit: 20,
     };
 
-    await fetchAllEntries(params);
+    const result = await fetchAllEntries(params);
 
     expect(mockClient.withoutLinkResolution.getEntries).toHaveBeenCalledTimes(5);
+    expect(result.includes.Entry).toHaveLength(6);
```

Comment:
It would be great to add a comment as to why we compare with 6, eg.

```
// Five pages will return those Entries
// Page 1: `some-entry-id` , `some-entry-id-0`
// Page 2: `some-entry-id` , `some-entry-id-20`
// Page 3: `some-entry-id` , `some-entry-id-40`
// Page 4: `some-entry-id` , `some-entry-id-60`
// Page 5: `some-entry-id` , `some-entry-id-80`
// As `result.includes.Entry` returns only unique entries, there are only 6 uniques as duplicates of `some-entry-id` are not returned
expect(result.includes.Entry).toHaveLength(6);
```